generator/TemperatureSensor.cs

```diff
@@ -0,0 +1,35 @@
+﻿using RabbitMQ.Client;
+using System;
+using System.Configuration;
+
+namespace generator
+{
+    class TemperatureSensor
+    {
+        private Random random;
+        private string queueName;
+        private IModel channel;
+        private int min;
+        private int max;
+        private int timeout;
+
+        public TemperatureSensor(IModel channel) 
+        {
+            this.random = new Random();
+            this.queueName = "temperature_sensor";
+            this.min = Int32.Parse(ConfigurationManager.AppSettings["MIN_TEMPERATURE"]);
```

Comment:
IDK how it works with console application, but you can use appsettings.json (at least in webapi project type). With this you can avoid neccesity of mapping values by using just:
```
_config.GetValue<String>("RabbitHost");
_config.GetValue<Int32>("RabbitPort")
```
when the config file is just simple json:
```
{
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "Microsoft.AspNetCore": "Warning"
    }
  },
  "AllowedHosts": "*",
  "RabbitUsername": "admin",
  "RabbitPassword": "admin",
  "RabbitPort": 5672,
  "RabbitHost": "rabbitmq",
  "QueueNameTemperature": "temperature_sensor"
}
```

Reply:
appsettings.json also works with console applications so I used it